src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -1309,13 +1309,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
     }
 
     /**
-     * DELETE /courses/:courseId/students/:studentLogin : Remove the given user from the students of the course so that the student cannot access the course any more
+     * DELETE courses/:courseId/students/:studentLogin : Remove the given user from the students of the course so that the student cannot access the course any more
```

Comment:
```suggestion
     * DELETE /courses/:courseId/students/:studentLogin : Remove the given user from the students of the course so that the student cannot access the course any more
```